### PR TITLE
Added testing to the API

### PR DESCRIPTION
**Pull request by Bracciata**

### CODE REVIEW

File: api/app.js [hunk 1]
```diff
@@ -58,6 +58,4 @@ app.get('/', async function(req, res) {
         res.json({error:"Failed to read URL from string"});
     }
 });
-let server = app.listen(8080, function() {
-    console.log('Server is listening on port 8080');
-});
+module.exports = app;
```
Comment (emma-hubka) [inline api/app.js]: no newline found at the end of file error

File: api/app.test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+});
+describe("GET different testing urls", () => {
+  test("It should respond with a score when passing google.come ", async () => {
+    const response = await request(app).get("/?url=google.com");
+    expect(response.body['issues']).toBeUndefined();
+    // This site's score may change over time.
+    expect(response.body['totalScore']).toBeDefined();
+    expect(response.statusCode).toBe(200);
+  });
+});
```
Comment (emma-hubka) [inline api/app.test.js]: no newline found at the end of file error

File: api/server.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+const app = require("./app");
+app.listen(8080, () => {
+  console.log("Server has started!");
+});
```
Comment (emma-hubka) [inline api/server.js]: no newline found at the end of file error

**Review by emma-hubka — CHANGES_REQUESTED** [review]

File: api/server.js (hunk 3 above)
Comment (Bracciata) [reply api/server.js]: Should be okay. Unsure why it is doing that on each file. I will re add and commit

**Review by emma-hubka — APPROVED** [review]

**Review by BracciaCJ — APPROVED** [review]
Looks good to me

**Pull request merged**